src/PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp

```diff
@@ -75,30 +99,51 @@ class IsotropicHomogeneous {
   IsotropicHomogeneous& operator=(const IsotropicHomogeneous&) = delete;
   IsotropicHomogeneous(IsotropicHomogeneous&&) = default;
   IsotropicHomogeneous& operator=(IsotropicHomogeneous&&) = default;
-  ~IsotropicHomogeneous() = default;
-
-  IsotropicHomogeneous(double incompressibility, double rigidity) noexcept;
-
-  /// The strain-stress relation that describes the material properties
-  template <typename DataType, typename Frame>
-  tnsr::II<DataType, Dim, Frame> stress(
-      const tnsr::ii<DataType, Dim, Frame>& strain,
-      const tnsr::I<DataType, Dim, Frame>& x) const noexcept;
+  ~IsotropicHomogeneous() override = default;
+
+  IsotropicHomogeneous(double bulk_modulus, double shear_modulus) noexcept;
+
+  /// The constitutive relation that characterizes the elastic properties of a
+  /// material
+  tnsr::II<DataVector, Dim> stress(const tnsr::ii<DataVector, Dim>& strain,
+                                   const tnsr::I<DataVector, Dim>& x) const
+      noexcept override;
+
+  /// The bulk modulus (or incompressibility) \f$K\f$
+  double bulk_modulus() const noexcept;
```

Comment:
I believe the consensus for analytic solutions was to remove these getters and have `operator==` be `friend`.

Reply:
I need these to compute analytic solutions such as #1374